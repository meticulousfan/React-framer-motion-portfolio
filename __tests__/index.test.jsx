/**
 * @jest-environment jsdom
 */

import React from "react";
import { render } from "@testing-library/react";

import Home from "../src/pages/index";

jest.mock("next/router", () => ({
  useRouter() {
    return {
      route: "/",
      pathname: "",
      query: "",
      asPath: ""
    };
  }
}));

describe("Forside", () => {
  test("Forside laster inn og kan vises", () => {
    const { home } = render(<Home />);
    expect(home).toMatchSnapshot();
  });
});
