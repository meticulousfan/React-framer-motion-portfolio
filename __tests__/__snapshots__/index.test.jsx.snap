// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forside Forside laster inn og kan vises 1`] = `undefined`;

exports[`Forside Om Meg eksisterer 1`] = `undefined`;
